src/test/java/tempest/ModuleViewTest.java

```diff
@@ -4,6 +4,7 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import tempest.Old.ModuleView;
```

Comment:
We should just ditch the legacy code and add new tests.

Reply:
I'll delete ModuleView and its test class now. But have you seen the new tests in the GUI test folder?